### PR TITLE
syncs rewards with custom ones

### PR DESCRIPTION
**Pull request by carmelloB**
Note: feel free to balance the values I set for possible items.

## About the PR

Changes the loot table of reward spawns on the new expeditions
## Why / Balance

because guns are too hard to find now.
## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

tweak: salvage loot pool has been changed to include custom items that were obtainable before merge

### CODE REVIEW

**Comment by dvir001** [comment]
Remove your SpaceStation14.sln

**Comment by FoxxoTrystan** [comment]
CrateSyndicateLightSurplusBundle is a meth for pe as i do like when good/rare weapons are only found in bluespace events and not expedition we need to focus on.

Expedition should be easy to handle and so loot should not be too "good"

**Comment by carmelloB** [comment]
dvir im not sure how to make an or group, should i just decrease the percent chance?

File: Resources/Prototypes/Procedural/salvage_loot.yml [hunk 1]
```diff
@@ -5,6 +5,30 @@
   loots:
     - !type:RandomSpawnsLoot
       entries:
+        #Custom shit, feel free to change up the values and remove stuff
```
Comment (Cheackraze) [inline Resources/Prototypes/Procedural/salvage_loot.yml:8]: you should put this below the regular spawn table and also some of these need to be much rarer or its going to cheapen it way too much

File: SpaceStation14.sln
Comment (Cheackraze) [inline SpaceStation14.sln:1]: this file needs to be removed from the PR, either reverted or ignored.

**Pull request closed without merging**